README.md

```diff
@@ -158,7 +158,7 @@ We recommend keeping calibrations for all robots in your organization in a commo
 [package's documentation](ur_calibration/README.md) for details.
 
 #### Quick start
-Once the driver is built and the **externalcontrol** URCap is installed and running on the robot, you are good
+Once the driver is built and the **externalcontrol** URCap is installed on the robot, you are good
 to go ahead starting the driver. (**Note**: We do recommend, though, to calibrate your robot first.)
```

Comment:
Nit, but: we're not actually calibrating the robot, are we? It's more exporting the already present calibration data and converting it to be usable with ROS tools.

Reply:
true...

Reply:
Fixed in 8c86b7e